airbyte-integrations/connectors/source-snowflake/unit_tests/test_incremental_streams.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+
+from airbyte_cdk.models import SyncMode
+from pytest import fixture
+from source_snowflake.source import IncrementalSnowflakeStream
+
+#TODO all tests
```

Comment:
Noting the TODOs here

Reply:
I am willing to add unittest from branch CDC. I cannot do it here because there are slight changes in the next PR